package.json

```diff
@@ -20,6 +20,7 @@
     "vue": "^2.6.6",
     "vue-class-component": "^6.0.0",
     "vue-i18n": "^8.10.0",
+    "vue-js-modal": "^1.3.31",
```

Comment:
Remember to update the LICENSE for the newly-added dependency

Reply:
> Add it here?
> https://github.com/apache/skywalking/blob/master/apm-dist/release-docs/LICENSE

It's for backend dependencies I think, frontend dependencies should be added here https://github.com/apache/skywalking/tree/master/apm-dist/release-docs/licenses/ui-licenses

Reply:
done